pkg/util/types.go

```diff
@@ -26,6 +26,7 @@ const (
 )
 
```

Comment:
It seems it's better to put types.go under common. There is no utility in this file